omegaconf/basecontainer.py

```diff
@@ -575,9 +577,12 @@ def _item_eq(
     def _is_none(self) -> bool:
         return self.__dict__["_content"] is None
 
-    def _is_missing(self) -> bool:
+    def _is_missing(self, throw_on_resolution_failure: bool = True) -> bool:
```

Comment:
What if you change this function to not throw on resolution failure (without a parameter), does this cause any issues?

Reply:
It doesn't cause any issue, It could be changed to that.